### DISCUSSION TITLE
What is this "websocket connection is closing" message I'm suddenly getting?

### DISCUSSION THREAD

**Discussion by redfoxofdeath** [Rapptz/discord.py#6343] [Q&A]
The other day my bot started spewing out "websocket connection is closing" to the console after being online for a few hours. The issues I've looked into with this message both mention shards (which I have no clue about, and my code doesn't mention that word even once). Apparently it has something to do with the aiohttp library, which I assume discord.py uses internally, but I don't use it in my code. Does anyone have know why this happens?

**Comment by Harmon758** [comment]
See https://github.com/Rapptz/discord.py/discussions/6045#discussioncomment-132275 and https://github.com/Rapptz/discord.py/discussions/6045#discussioncomment-139574.

**Reply by redfoxofdeath** [reply]
Interesting that it has been removed. Perhaps I'll be satisfied with that
Thank you
